cg/services/illumina/backup/backup_service.py

```diff
@@ -278,37 +286,76 @@ def retrieve_archived_file(self, archived_file: Path, run_dir: Path) -> None:
             file_path=str(archived_file), target_path=str(retrieved_file)
         )
 
+    @staticmethod
+    def parse_dsmc_output_sequencing_path(dsmc_output: list[str]) -> list[PdcSequencingFile]:
+        """Parses the DSMC command output to extract validated sequencing paths."""
+        validated_responses = []
+        for line in dsmc_output:
+            if is_pdc_sequencing_path(line):
+                parts = line.split()
+
+                fileDateTime: datetime = convert_string_to_datetime_object(
+                    f"{parts[DsmcOutput.DATE_COLUMN_INDEX]} {parts[DsmcOutput.TIME_COLUMN_INDEX]}"
+                )
+
+                query_response: PdcSequencingFile = PdcSequencingFile(
+                    dateTime=fileDateTime,
```

Comment:
```suggestion
                file_date_time: datetime = convert_string_to_datetime_object(
                    f"{parts[DsmcOutput.DATE_COLUMN_INDEX]} {parts[DsmcOutput.TIME_COLUMN_INDEX]}"
                )

                query_response: PdcSequencingFile = PdcSequencingFile(
                    date_time=file_date_time,
```

Reply:
The pythonic convention is to use snake case for variables. If this is due to external software using a different convention, we can find a workaround